app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,8 @@
+<li id="post-<%= post.id %>">
```

Comment:
- [ ] Post is always displayed with content and author
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts4.png)

-- Let's add the post author.